src/components/Navigation.css

```diff
@@ -0,0 +1,24 @@
+.navigation ul {
+  padding: 0;
+  display: flex;
```

Comment:
You guys did a good job. One suggestion I would like is to add flex-direction to column so that our list can appear in block.